SS_readdata_330.tpl

```diff
@@ -678,7 +678,7 @@
 
   data_type = 1; //  for surveys
   echoinput << "Units:  0=numbers; 1=biomass; 2=F; >=30 for special patterns" << endl;
-  echoinput << "Errtype:  -1=normal; 0=lognormal; >0=T" << endl;
+  echoinput << "Errtype:  -2=gamma(future); -1=normal; 0=lognormal ; 1=lognormal w/ biascorr; >1XXX=T-dist with DF=XXX" << endl;
```

Comment:
df=1 is a cauchy which probably we never want to use, so why not let Errtype>1 be t-dist with df=Errtype?

Reply:
Actually a potentially easier implementation to leave the t-dist alone is to just use -2 for lognormal bias corr and I'll use -3 for gengamma later. Then you don't have to change anything related to t-dist, I think...